src/common/exception/src/exception_code.rs

```diff
@@ -145,6 +145,7 @@ build_exceptions! {
     InvalidTimestamp(1080),
     InvalidClusterKeys(1081),
     UnknownFragmentExchange(1082),
+    InvalidUtf8(1083),
```

Comment:
I don't think it's a meaning error code that should be returned to users. How about use `InternalError`?